extern/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 #  iDynTree
 #  Copyright: (C) 2016 iCub Facility, Istituto Italiano di Tecnologia
-#  Authors: Silvio Traversaro <silvio.traversaro@iit.it>
+#  Authors: Silvio Traversaro <silvio.traversaro@iit.it> 
```

Comment:
```suggestion
#  Authors: Silvio Traversaro <silvio.traversaro@iit.it>
```

Please try to avoid adding  trailing whitespaces. You can configure most editors to automatically remove them.